Add PagerDuty plugin initial tables, docs. Closes #1 (#2)
turbot/steampipe-plugin-pagerduty: docs/tables/pagerduty_priority.md

# Table: pagerduty_priority

>  A priority is a label representing the importance and impact of an incident. This feature is only available on `Standard` and `Enterprise` plans.

Reviewer: ```suggestion
A priority is a label representing the importance and impact of an incident. This feature is only available on `Standard` and `Enterprise` plans.
```

>  Incident priority levels help you classify the most important incidents from the least important ones. Teams can quickly see which incidents need their immediate attention. If your organization already have priority levels defined, you can customize these default values to match your existing incident classification scheme.

Reviewer: ```suggestion
Incident priority levels help you classify the most important incidents from the least important ones. Teams can quickly see which incidents need their immediate attention. If your organization already have priority levels defined, you can customize these default values to match your existing incident classification scheme.
```

>  **Note:** To list the priorities, first [enable and configure your priorities](https://support.pagerduty.com/docs/incident-priority#section-enabling-incident-priority).

Reviewer: ```suggestion
**Note:** In order to list priorities, an account owner will need to [enable incident priority levels](https://support.pagerduty.com/docs/incident-priority#section-enabling-incident-priority).
```

## Examples

### Basic info

```sql
select
  name,
  id,
  description,
  self
from
  pagerduty_priority;
```

> ### List event rules with high-priority(P1)

Reviewer 2: ```suggestion
### List event rules with highest-priority(P1)
```

> ### List event rules with highest-priority(P1)

Reviewer: ```suggestion
### List event rules with highest priority (P1)
```
